tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -204,9 +206,10 @@ Future<void> _main(List<String> args) async {
   parser.addCommand(cmdAnalyzePubspecs, analyzePubspecs);
 
   ArgResults results;
+  var devPrint = true;
   try {
     results = parser.parse(args);
-    bool devPrint = results['development-print'];
+    devPrint = results['development-print'];
```

Comment:
Chore: This has more values than just development-prints now. We should refactor the meaning of the variable so that it silences all prints. Will do that as part of solving this issue: https://github.com/serverpod/serverpod/issues/1042